bodhi/client/__init__.py

```diff
@@ -985,6 +985,90 @@ def info_release(name, url, **kwargs):
         print_release(res)
 
 
+@releases.group(name='critpath')
+def critpath_pkgs():
+    # Docs that show in the --help
+    """Interact with critical path packages of a release."""
+    # Developer Docs
+    """Manage the critical path packages of a release."""
+    pass  # pragma: no cover
+
+
+@critpath_pkgs.command(name='list')
+@handle_errors
+@click.argument('name')
+@url_option
+@staging_option
+def list_critpath_packages(name, url, **kwargs):
+    """
+    List critical path packages of a release.
+
+    NAME: Release name (e.g. F27)
+    """
+    client = bindings.BodhiClient(base_url=url, staging=kwargs['staging'])
+
+    res = client.send_request('releases/{}/critpath-packages'.format(name), verb='GET', auth=False)
+
+    if 'errors' in res:
```

Comment:
If you make a binding for getting critical path packages, you can wrap it with ```bodhi.client.bindings.errorhandled()``` and then you don't need to do this because it will do it for you. This will also help releng who will want to refactor their script to use the bindings (not the CLI).